tools/lint.py

```diff
@@ -116,9 +116,10 @@ def main():
     else:
         files = args.files
 
-    cython_exts = ['.pyx']  # Add '.pxd', '.pxi' once cython-lint supports it
-    cython_files = [f for f in files if any(f.endswith(ext) for ext in cython_exts)]
-    other_files = [f for f in files if not f.endswith('.pyx')]
+    cython_exts = ('.pyx', '.pxd', '.pxi')
+    # Add '.pxd', '.pxi' once cython-lint supports it
+    cython_files = [f for f in files if f.endswith('.pyx')]
```

Comment:
The Cython linter supports these files now, so I'd leave this line as it was and fix up the few errors reported, if any.

Reply:
Updated to include `pxd` and `pxi` files